backend/src/scraper/bestbuy_scraper.py

```diff
@@ -51,15 +51,22 @@ def _product_specs(product_id: str) -> Dict:
                     { "name": s.get("name"), "value": s.get("value") } for s in spec_highlights
                 ],
             }
+        
+    @staticmethod    
+    def _pull_reviews(i: int, sku: int, params: Dict) -> List[str]:
+        search_sku = requests.get(f"https://www.bestbuy.ca/api/v2/json/reviews/{sku}?page={i}&source=us", headers = params.get('headers'))
+        review_results = search_sku.json()
+        reviews = review_results.get("reviews", [])
+        return reviews
 
     @staticmethod
-    def _product_reviews(product_id: str, product_name: str) -> Dict:
-        # Search for product by UPC.
+    def _product_reviews(product_name: str) -> Dict:
+        # Search for product by name (first 5 words).
         params = {
-            "product_id": product_id,
+            "product_name": ' '.join(product_name.split()[:5]).lower(),
```

Comment:
Just curious, why only first 5 words of product name?

Reply:
Because the product titles are super long and descriptive and I think that there are really only 2 pieces of information that are truly important: brand and model. 5 words accounts for brand, model, and any other filler words (like a dash, "camera", "DSLR", etc.)